editor/editor_command_palette.cpp

```diff
@@ -111,7 +110,7 @@ void EditorCommandPalette::_update_command_search(const String &search_text) {
 			}
 
 			TreeItem *ti = search_options->create_item(section);
-			String shortcut_text = entries[i].shortcut_text == "None" ? "" : entries[i].shortcut_text;
+			String shortcut_text = entries[i].shortcut_text == "None" ? "" : entries[i].shortcut_text + "  ";
```

Comment:
I assume you added this because of the low separation width in the right border. That is due to a regression in `Tree`, so this is unnecessary.

Reply:
yes, okay I will remove it.